src/Compilers/Test/Resources/Core/Microsoft.CodeAnalysis.Compiler.Test.Resources.csproj

```diff
@@ -3,7 +3,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
Should we be using `RoslynPortableTargetFrameworks`, so we don't have to update these manually when we change that?

Reply:
https://github.com/dotnet/roslyn/blob/master/eng/targets/Settings.props#L19

Reply:
This appears to be a different value than what is used in our library projects, including [Microsoft.CodeAnalysis.CSharp](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj#L8) for example.

Reply:
I would be open to adding some build variable to hold `netcoreapp3.1;netstandard2.0` though.

Reply:
We probably should, perhaps `RoslynPortableTestFrameworks`?